spec/v1/CSN-Interop-Effective.schema.yaml

```diff
@@ -1069,19 +1069,20 @@ definitions:
       "on": &on
         type: array
         description: |-
-          The property `on` holds a sequence of operators and operands to describe the join condition.
+          The property `on` holds a sequence of operators and operands to describe the join condition, similar to an SQL expression.
 
-          One building block of the sequence consists of the following in the given order:
-          - Reference to a field of the association or composition target
+          The `on` condition is constructed by triples of:
+          - Reference to the target element (ID) as array with 2 items 
           - Equals Operator "="
-          - One of the following:
-            - Reference to a field of the source entity
-            - Constant Value
+          - Reference to the local element (ID)as array with 1 item OR a constant value (`val`)
 
-          This building block states that the value of the first entry of the array using "ref" should equal the value of the third entry of the array.
-          In addition, several building blocks can be lined up using an "and" operator.
+          The first and third entry MAY be reversed but the `=` operator MUST be in the middle.
+          The target element reference MUST have two array items. The first item is the association name and the second item is the target element name.
```

Comment:
Do we need further clarification on this? It was misunderstood to contain the target entity name in one case.

Reply:
I think with the "(not the target entity name!)" from above it should be clear